lib/mapbox/layer-factory.js

```diff
@@ -10,15 +10,7 @@ export default function layerFactory(sourceType, layer) {
 }
 
 function buildGeoJSONLayer(layer) {
-  const source = {
-    type: 'geojson',
-    data: {
-      type: 'FeatureCollection',
-      features: [],
-    },
-  }
-
-  return merge({ source }, layer)
+  return layer
```

Comment:
remove function?

Reply:
fix in: Feat-GDV-196-split-vector-layers-and-configuration